### DISCUSSION TITLE
Design Resources Repo (React Components)

### DISCUSSION THREAD

**Discussion by vixalien** [vercel/vercel#4327] [General] [answered]
At [vercel.com/design](vercel.com/design) there are many components, and I recently wanted to use them.

Are they open source?

Can I get their React source.

**PS**: I tried checking: `nextjs-site` and `docs` repositories but they don't look like the components specified on the [vercel.com/design](vercel.com/design) page

**Comment by roelvan** [comment]
They are not open source AFAIK.

**Comment by styfle** [comment, marked as answer]
These components are not open source at this time but will be in the future.

No ETA yet but @pacocoursey is working on it.
